content/en/Platform Deep Dive/Assets/_index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Assets"
+linkTitle: "Assets"
+weight: 10
+description: >
+  Assets are _what_ we pentest.
+---
+
+{{% pageinfo %}}
+Our security professionals perform pentests on different types of assets. Learn how to create and manage your assets.
+{{% /pageinfo %}}
+
+{{% asset-definition %}} Once you've created an asset, you can launch pentests for it.
+
+## Asset Types
+
+We can perform pentests on assets in the following categories:
+
+{{% asset-types-table %}}
+
+For assets of multiple types, you may consider to set up one or more pentests, depending on the characteristics of your software.
+
+{{%expand "Learn more." %}}
+
+Your asset combines a web and mobile application. You may want to test them together, in one pentest, if:
+
+- The two applications share some of the same code and functionalities.
```

Comment:
@TheInstaGrahame Does this make sense? Do we accept pentests if a Web/Mobile app share only a little bit of the same code? (How rigorous are we with the distinction?)

```suggestion
- The two applications share most of the same code and functionalities.
```

Reply:
> @TheInstaGrahame Does this make sense? Do we accept pentests if a Web/Mobile app share only a little bit of the same code? (How rigorous are we with the distinction?)

I relied on this Zendesk article > FAQ > Question 2: https://zombie.atlassian.net/wiki/spaces/DES/pages/2468512531/Pendo+Design+Guidelines

Reply:
Since we're still waiting on @TheInstaGrahame , I'm going to approve this PR. If Grahame wants a change, we'll open a new PR.